lumicks/pylake/file.py

```diff
@@ -53,24 +52,24 @@ def _check_file_format(self):
         if ff_version not in File.SUPPORTED_FILE_FORMAT_VERSIONS:
             raise Exception(f"Unsupported Bluelake file format version {ff_version}")
 
-        # List of fields which should not be accessed directly from the h5 and their actual API accessors. Note that
-        # top level fields are automatically printed when print is invoked.
-        self.redirect_list = OrderedDict(
-            [
-                ("Calibration", "force1x.calibration"),
-                ("Marker", "markers"),
-                ("FD Curve", "fdcurves"),
-                ("Kymograph", "kymos"),
-                ("Scan", "scans"),
-                ("Note", "notes"),
-            ]
-        )
+        # List of fields which should not be accessed directly from the h5 and their actual API
+        # accessors. Note that top level fields are automatically printed when print is invoked.
```

Comment:
maybe add to this the new behavior (only warns if the output class isn't defined) also maybe add a note about the output class (2nd item in tuple) I didn't really know immediately what it was there for